helpers/env-variables.ts

```diff
@@ -293,7 +291,6 @@ const getModelEnvs = (modelConfig: ModelConfig): EnvVar[] => {
           {
             name: "GOOGLE_API_KEY",
             description: "The Google API key to use.",
-            value: modelConfig.apiKey,
```

Comment:
@leehuwuj how does this approach generate the key in the env file if the user is explicitly setting it?